encryptor/config/encryptionSettings.go

```diff
@@ -230,38 +230,43 @@ func (s *BasicColumnEncryptionSetting) Init() (err error) {
 	if s.ReEncryptToAcraBlock != nil && *s.ReEncryptToAcraBlock {
 		s.settingMask |= SettingReEncryptionFlag
 	}
+	var tokenType common.TokenType
+	var ok bool
 	if s.TokenType != "" || s.Tokenized {
-		tokenType, ok := tokenTypeNames[s.TokenType]
+		tokenType, ok = tokenTypeNames[s.TokenType]
```

Comment:
Here we define `ok` in scope of the entire function, though it is used only in scope of this `if` statement. And them, the `ok` is overwritten on 297th line. For the reader this is confusing, because my first assumption was that the result of this variable is used somewhere below.